src/Bravo3/Orm/Services/Io/Reader.php

```diff
@@ -56,9 +57,39 @@ public function getPropertyValue($name)
         } elseif ($relationship = $this->metadata->getRelationshipByName($name)) {
             $getter = $relationship->getGetter();
             return $this->entity->$getter();
-        } else {
-            throw new InvalidArgumentException("No column/relationship found for property '".$name."'");
```

Comment:
Changing logic here.. better to wrap upstream code in a `try..catch(InvalidArgumentException)`?